src/clarity-angular/main.scss

```diff
@@ -7,52 +7,80 @@
 
 /* @echo VERSION */
 
-
+/** Order of imports is important **/
 @import "./utils/overwrites.clarity";
-@import "./utils/mixins.clarity";
-@import "./utils/normalize.clarity";
-@import "./utils/reboot.clarity";
-@import "./utils/helpers.clarity";
-@import "./utils/layers.clarity";
+@import "./utils/layers.clarity"; // no dependencies on other clarity scss
+@import "./utils/helpers.clarity"; // no dependencies on other clarity scss
+@import "./color/color.clarity"; // depends on colors.clarity, helpers.clarity, contrast-cache.clarity
```

Comment:
If `mixins.clarity` always imports `color.clarity` inside it, is it necessary to import it again in this `main.scss`? Or is this so that the dependency is more explicit and clear? I don't think there's harm in importing twice, since we export style so that they are not included twice in the compiled css, but just noting it here.

I guess in general maintaining this 'master list' of imports is kind of a pain and error-prone. I don't have an alternative solution at the moment, but maybe something we can evaluate again in the future.